modules/ROOT/pages/queries/select-version.adoc

```diff
@@ -0,0 +1,252 @@
+=== Procedures and functions
+
+link:{neo4j-docs-base-uri}/operations-manual/current/procedures[Procedures] and xref:functions/index.adoc[functions] (including built-in and link:{neo4j-docs-base-uri}/apoc/current/[APOC]) are tied to a specific Cypher language version.
+Therefore, procedures and functions in Neo4j 2025.03+ and APOC 2025.03+ (both of which have Cypher 25 as their default language) may behave differently depending on what version of Cypher is used.
+
+For example, APOC 2025.03 removed Cypher 25 support of the procedure `apoc.create.uuids()`, meaning it is not available to queries running Cypher 25.
+However, it can still be used on APOC 2025.03 if queries are prepended with `CYPHER 5`, or if the database’s default version is set to `CYPHER 5`.
+In this case, Neo4j will use APOC and Cypher as they existed at the time of the 5.26 LTS release.
+
+.Using a procedure removed in Cypher 25 with APOC 2025.03+
+[source, cypher]
+----
+CYPHER 5
+CALL apoc.create.uuids(10)
+----
+
+[[cypher-selection-with-other-query-options]]
+=== Combine Cypher version selection with other query options
+
+It is possible to combine Cypher version selection with other xref:planning-and-tuning/query-tuning.adoc[query options].
+The below example selects both the version and the xref:planning-and-tuning/runtimes/concepts.adoc[runtime] of Cypher for the same query:
+
+.Combining Cypher version selection with other query options
+[source, cypher]
+----
+CYPHER 5 runtime=parallel
+MATCH (n:Person)
+RETURN n.name
+----
+
+[[select-default-cypher-version]]
+== Select the default Cypher version when creating a database
+
+Databases created on Neo4j 2025.03 or later will have Cypher 25 as their default language (if {neo4j-docs-base-uri}/operations-manual/current/configuration/configuration-settings/#config_db.query.default_language[`config_db.query.default_language`] is set to `CYPHER_25` -- see  xref:queries/select-version.adoc#config-database-upgrades[Cypher versions, configuration settings, and DBMS upgrades] below).
+This is true for the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/#manage-database-systems[system, standard, and composite] Neo4j databases.
+
+To select a different default Cypher version for a database, add `DEFAULT LANGUAGE <language version>` to the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/standard-databases/create-databases/[`CREATE DATABASE`] statement.
+This can be done on any Neo4j database.
+
+Selecting `CYPHER 5` as the default database language ensures that every query run on that database uses the language as it existed at the time of the Neo4j 5.26 LTS release (unless a query is prepended with `CYPHER 25`, which overrides this default).
+Any changes introduced after 5.26 will not affect the semantics of the query.
+
+.Create a database with Cypher 5 as the default language
+[source, cypher]
+----
+CREATE DATABASE my_database DEFAULT LANGUAGE CYPHER 5
```

Comment:
I have linked back to these sections from the operations manual documentation of these commands. There doesn't seem much point documenting it twice.

Reply:
I think all sections related to database administration should go to the Operations manual, and here, we can add just one link.